gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -1,98 +1,134 @@
+          <div class="row form-group">
+            <div class="col-md-6">
+              <div class="form-group">
+                <label translate for="addresses">Addresses</label>
+                <input ng-disabled="currentDevice.deviceID == myID" id="addresses" class="form-control" type="text" ng-model="currentDevice._addressesStr"></input>
+                <p translate class="help-block">Enter comma separated ("tcp://ip:port", "tcp://host:port") addresses or "dynamic" to perform automatic discovery of the address. </p>
+              </div>
+            </div>
+            <div class="col-md-6">
+              <div class="form-group">
+                <label translate>Compression</label>
+                <select class="form-control" ng-model="currentDevice.compression">
+                  <option value="always" translate>All Data</option>
+                  <option value="metadata" translate>Metadata Only</option>
+                  <option value="never" translate>Off</option>
+                </select>
+              </div>
             </div>
           </div>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col-md-12">
-          <div class="form-group">
-            <label translate for="folders">Share Folders With Device</label>
-            <p translate class="help-block">Select the folders to share with this device.</p>
-            <div class="row">
-              <div class="col-md-4" ng-repeat="folder in folderList()">
-                <div class="checkbox">
-                  <label ng-if="folder.label.length == 0">
-                    <input type="checkbox" ng-model="currentDevice.selectedFolders[folder.id]">&nbsp;{{folder.id}}
-                  </label>
-                  <label ng-if="folder.label.length != 0">
-                    <input type="checkbox" ng-model="currentDevice.selectedFolders[folder.id]">&nbsp;<span tooltip data-original-title="{{folder.id}}">{{folder.label}}</span>
-                  </label>
+          <div class="row form-group">
+            <div class="col-md-12">
+              <label translate>Device rate limits</label>
+              <div class="row">
+                <div class="col-md-6">
+                  <div ng-class="{'has-error': deviceEditor.maxSendKbps.$invalid && deviceEditor.maxSendKbps.$dirty}">
+                    <div class="row">
+                      <span class="col-md-8" translate>Outgoing Rate Limit (KiB/s)</span>
+                      <div class="col-md-4">
+                        <input name="maxSendKbps" id="maxSendKbps" class="form-control" type="number" ng-model="currentDevice.maxSendKbps" required="" aria-required="true" min="0"/>
+                      </div>
+                    </div>
+                    <p class="help-block" ng-if="!deviceEditor.maxSendKbps.$valid && deviceEditor.maxSendKbps.$dirty" translate>
+                      The rate limit must be a non-negative number (0: no limit) </p>
```

Comment:
The way this is broken over multiple lines makes me sad, please adjust to either `<p>foo</b>` or
```
<p>
   Foo
</p>
```